<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Apprentice Video</string>
  </property>
  <property name="windowIcon">
   <iconset resource="apprenticevideo.qrc">
    <normaloff>:/images/aeyae-remux-logo.png</normaloff>:/images/aeyae-remux-logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>13</number>
   </property>
   <property name="verticalSpacing">
    <number>11</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="spacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>238</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="spacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="apprenticevideo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>AboutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
